oak_proto_rust/Cargo.toml

```diff
@@ -6,6 +6,7 @@ license = "Apache-2.0"
 
 [dependencies]
 prost = { workspace = true }
+zeroize = "*"
```

Comment:
Assigning zeroes is not sufficient (that's why the zeroize crate exists). Cleaning up at the call site makes sense, but here the risk of leaving around secrets in memory is too high, so I think this dependency is allowed. Also I imagine we will probably need it in all layers, since each of them will need to at least zero out its own DICE secret

Reply:
I moved zeroize code into the actual function that parses DICE data for now. Once we decide that we can use `zeroize` in `oak_proto_rust`, I'll add it in a separate PR

Reply:
Understood - either option LGTM then, thanks!